intro-to-oop-contd/projects-with-many-files.md

```diff
@@ -0,0 +1,295 @@
+# Projects with Many Files
+
+## Learning Goals
+
+- Use Python syntax to "link" two files together
+- Demonstrate how code jumps between files when working with multiple classes
+- Apply lessons about OOP to managing several classes in one project
+
+## Introduction
+
+Scarlet is making a ride share app. In her project, she has two classes, `Driver` and `Passenger`. She starts her project in one file, `main.py`. It's important that each `Passenger` instance has an attribute `favorite_new_driver`, which is an instance of `Driver`. Her code could look like:
+
+```python
+class Driver:
+
+    def __init__(self):
+        print("I'm creating an instance of Driver!")
+
+class Passenger:
+
+    def __init__(self):
+        print("I'm creating an instance of Passenger!")
+        self.favorite_new_driver = Driver()
+
+scarlet = Passenger()
+```
+
+Even though we can define multiple classes in one `.py` file as long as we have the correct syntax, we may want to re-organize our code.
+
+It's conventional to have a separate `.py` file for each class defined. However, what are the other steps we'll need to take to support that organization?
+
+The organization style that this lesson teaches is a recommendation!
+
+## Vocabulary and Synonyms
+
+| Vocab | Definition | Synonyms | How to Use in a Sentence
+| --- | --- | --- | ---
+
+## New File Organization
+
+Our projects so far contain source code, unit tests, git files, `README` files, configuration files (such as `requirements.txt`), etc. Now, with OOP, we're motivated to add to that; ideally, each class definition will have their own `.py` file. Consistent organization and management of these files can help make our code more navigable and discoverable. What kind of guidelines for folders and files can we follow?
```

Comment:
I think my thoughts are:

1. I appreciate and agree with decreasing the use of semicolons overall! It's something I'll continuously work on
2. For the process of refactoring them out, I'm personally happy with keeping a semicolon one out of every ten lessons (this is a random ratio). If it takes longer than four minutes to come up with a better way to word things, then let's keep the semicolon!

For this specific case, I feel like we could replace this semicolon with a `.` or `!` and it'll work okay, personally.

Reply:
Well, according to my deep lore, your use of semicolons derives from a deep and abiding belief that concepts want to be related and linked with one another, and not sequestered by the harsh divisions of full stops.  In general, I'll back off quashing them, and in the end I expect this will probably be a non-issue.